pkg/ext/apiserver_authentication_test.go

```diff
@@ -12,22 +13,32 @@ import (
 
 	"github.com/stretchr/testify/require"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metainternalversion "k8s.io/apimachinery/pkg/apis/meta/internalversion"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apiserver/pkg/authentication/authenticator"
 	"k8s.io/apiserver/pkg/authentication/user"
 	"k8s.io/apiserver/pkg/authorization/authorizer"
 	"k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/apiserver/pkg/registry/rest"
 	"k8s.io/apiserver/pkg/server/options"
 )
 
+var _ rest.Storage = (*authnTestStore)(nil)
+var _ rest.Lister = (*authnTestStore)(nil)
```

Comment:
It's a matter of stylistic preference but I would do
```golang
var _ rest.Storage = &authnTestStore{}
var _ rest.Lister = &authnTestStore{}
```
Allocations should not be happening, if this is a concern.

Reply:
I'll leave as is since it's mostly stylistic. The codebase already uses (*<thing>)(nil) in a few places so I don't see a reason to change it.